### PR TITLE
Update the base Docker image

### PR DESCRIPTION
**Pull request by cbaines**
### What
Update the base Docker image

This will upgrade the ruby and freeradius versions, plus updating
other stuff as well.

### Why
Switching to Ruby 3 would be good.

Link to Trello card: https://trello.com/c/7NaQqcb3/2040-bump-govwifi-frontend-ruby-and-alpine-version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # WARNING: We must pin the Alpine version to 3.8 because freeradius-rest (r6) package in alpine-3.9 contains
```
Comment (sarahseewhy) [inline Dockerfile:1]: We can remove the comment as it no longer applies, I wouldn't want it to confuse people in the future.

**Review by sarahseewhy — APPROVED** [review]
LGTM, just remove the unnecessary comment.

Also worth double checking the FreeRADIUS [Release Notes](https://freeradius.org/release_notes/) to ensure there aren't any other breaking changes between the versions.

File: Dockerfile (hunk 1 above)
Comment (cbaines) [reply Dockerfile]: Done

**Comment by cbaines** [comment]
> Also worth double checking the FreeRADIUS [Release Notes](https://freeradius.org/release_notes/) to ensure there aren't any other breaking changes between the versions.

I've had a read through the release notes now.

I've actually made one change off the back of that, tweaking the `cipher_list` configuration since we're currently using the "DEFAULT" ciphers, and I think keeping currently enabled ciphers enabled might be required to not cause connectivity issues like before.

**Comment by sarahseewhy** [comment]
Great! Thank you for double checking. Let's carefully roll this out and do as much testing as we can in staging.

**Comment by cbaines** [comment]
> I've actually made one change off the back of that, tweaking the cipher_list configuration since we're currently using the "DEFAULT" ciphers, and I think keeping currently enabled ciphers enabled might be required to not cause connectivity issues like before.

I'm flip flopping, but I'm going back on this change. From the fragmented information I was putting together earlier, I think this SECLEVEL was documented specifically in relation to EAP-FAST, which we're not using. Additionally, it doesn't seem to make any difference to the enabled ciphers. I've gone back to not specifying it now.

I've been looking in to the effects of the OpenSSL upgrade (1.0.2u -> 1.1.1l) on the supported ciphers, and I've included a list of the ones which this change will remove support for. Hopefully there's no clients that are relying on ciphers in this list and are unable or configured not to negotiate alternatives. 

```
CAMELLIA128-SHA
CAMELLIA256-SHA
DES-CBC3-SHA
DH-DSS-CAMELLIA128-SHA
DH-DSS-CAMELLIA256-SHA
DH-DSS-DES-CBC3-SHA
DH-DSS-SEED-SHA
DHE-DSS-CAMELLIA128-SHA
DHE-DSS-CAMELLIA256-SHA
DHE-DSS-SEED-SHA
DHE-RSA-CAMELLIA128-SHA
DHE-RSA-CAMELLIA256-SHA
DHE-RSA-SEED-SHA
DH-RSA-CAMELLIA128-SHA
DH-RSA-CAMELLIA256-SHA
DH-RSA-DES-CBC3-SHA
DH-RSA-SEED-SHA
ECDH-ECDSA-DES-CBC3-SHA
ECDH-ECDSA-RC4-SHA
ECDHE-ECDSA-DES-CBC3-SHA
ECDHE-ECDSA-RC4-SHA
ECDHE-RSA-DES-CBC3-SHA
ECDHE-RSA-RC4-SHA
ECDH-RSA-DES-CBC3-SHA
ECDH-RSA-RC4-SHA
EDH-DSS-DES-CBC3-SHA
EDH-RSA-DES-CBC3-SHA
IDEA-CBC-SHA
PSK-3DES-EDE-CBC-SHA
PSK-RC4-SHA
RC4-MD5
RC4-SHA
SEED-SHA
SRP-3DES-EDE-CBC-SHA
SRP-DSS-3DES-EDE-CBC-SHA
SRP-DSS-AES-128-CBC-SHA
SRP-DSS-AES-256-CBC-SHA
SRP-RSA-3DES-EDE-CBC-SHA
```

**Review by sarahseewhy — APPROVED** [review]
Thanks for doing that investigation into openssl.

Happy to approve this and then roll it out next week.

**Pull request merged**